crates/utils/src/tracing/grpc.rs

```diff
@@ -1,6 +1,6 @@
 /// Creates a [`tracing::Span`] based on RPC service and method name.
 macro_rules! rpc_span {
-    ($service:expr, $method:expr) => {
+    ($service:expr_2021, $method:expr_2021) => {
```

Comment:
This can probably be a `literal` since I think its exclusively used with string literals.

```suggestion
    ($service:literal, $method:literal) => {
```